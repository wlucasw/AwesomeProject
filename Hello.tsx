import React from 'react';
import {StyleSheet, Text, View} from 'react-native';

const styles = StyleSheet.create({
  green: {
    color: 'green',
    fontSize: 30,
  },
});

const Hello = () => {
  return (
    <View
      style={{
        flex: 1,
        justifyContent: 'center',
        alignSelf: 'center',
        alignItems: 'center',
        width: 150,
        height: 150,
        backgroundColor: 'red',
        margin: 10,
      }}>
      <Text style={styles.green}>Hello !</Text>
    </View>
  );
};

export default Hello;
