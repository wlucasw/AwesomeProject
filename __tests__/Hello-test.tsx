import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Hello from '../Hello';

test('renders correctly', () => {
  const tree = renderer.create(<Hello />).toJSON();
  expect(tree).toMatchSnapshot();
});
