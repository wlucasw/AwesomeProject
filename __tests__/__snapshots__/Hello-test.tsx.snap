// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "red",
      "flex": 1,
      "height": 150,
      "justifyContent": "center",
      "margin": 10,
      "width": 150,
    }
  }
>
  <Text
    style={
      Object {
        "color": "green",
        "fontSize": 30,
      }
    }
  >
    Hello !
  </Text>
</View>
`;
